test/unit/Injector.spec.js

```diff
@@ -92,19 +91,19 @@ describe('Injector', function () {
 
   it('dont allow async use of $injector', (done) => {
     this.injector.registerDependencies({
-      _: 'lodash',
-      chai: 'chai'
```

Comment:
These are fake dependencies that resolve to a string, so it was confusing why they were named after real packages or dependencies that were not dependencies.